src/components/AddItem.js

```diff
@@ -59,7 +66,7 @@ const AddItem = () => {
         });
       } else {
         setHasError(true);
-        setTimeout(() => setHasError(false), 3000);
+        setTimeout(() => setHasError(false), 30000);
```

Comment:
I'm curious about this change, why is the timeout changed from 3s to 30s?

Reply:
My bad, @llunaplanet!
I was adding it to fix the error for accessibility issue.
Will fix it back to 3000. 
Thank you for the catch! 😄